paasta_tools/cli/cmds/mark_for_deployment.py

```diff
@@ -65,9 +67,11 @@ def add_subparser(subparsers):
 
 def mark_for_deployment(git_url, cluster, instance, service, commit):
```

Comment:
Can you... take this code and use it in the itest code that "mocks" it?

(general_itests/steps/deployments_json_steps.py:56)